app/views/declaration/link_ducr_to_mucr.scala.html

```diff
@@ -0,0 +1,102 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import controllers.declaration.routes
+@import controllers.navigation.Navigator
+@import forms.common.YesNoAnswer
+@import forms.common.YesNoAnswer.YesNoAnswers
+@import forms.declaration.LinkDucrToMucr
+@import models.requests.JourneyRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+@import views.helpers.{BackButton, Title}
+@import views.components.gds.Styles._
+@import views.helpers.ErrorMapper.yesNoErrors
+@import views.html.components.gds._
+
+@this(
+    govukLayout: gdsMainTemplate,
+    govukDetails : GovukDetails,
+    yesNoRadios: yesNoRadios,
+    errorSummary: errorSummary,
+    sectionHeader: sectionHeader,
+    paragraph: paragraphBody,
+    link: link,
+    saveButtons: saveButtons,
+    saveAndContinue: saveAndContinue,
+    formHelper: FormWithCSRF,
+    appConfig: AppConfig
+)
+
+@(mode: Mode, form: Form[YesNoAnswer])(implicit request: JourneyRequest[_], messages: Messages)
+
+@mucrDetails = {
+    @paragraph(
+        id = Some("link-ducr-to-mucr-hint1"),
+        message = messages("declaration.linkDucrToMucr.details.hint1", link(
+            text = messages("declaration.linkDucrToMucr.details.hint1.link"),
+            call = Call("GET", appConfig.notesForMucrConsolidation),
+            target = "_blank"
+        ))
+    )
+
+    @paragraph(
+        id = Some("link-ducr-to-mucr-hint2"),
+        message = messages("declaration.linkDucrToMucr.details.hint2", link(
+            text = messages("declaration.linkDucrToMucr.details.hint2.link"),
+            call = Call("GET", appConfig.arriveOrDepartExportsService),
+            target = "_blank"
+        ))
+    )
+}
+
+@buttons = @{
+  request.declarationType match {
+    case DeclarationType.CLEARANCE => saveButtons()
+    case _                         => saveAndContinue()
+  }
+}
```

Comment:
Could you tell me why only on Clearance user is presented with 'Save for later' link/button?